# This is a false positive (cf https://wiki.debian.org/Hardening#Validation)
python3-llfuse: hardening-no-fortify-functions

